docs/Installing.asciidoc

```diff
@@ -19,6 +19,36 @@ assumed that the reader is already familiar with openQA and has already read the
 Starter Guide, available at the
 https://github.com/os-autoinst/openQA[official repository].
 
+== openQA quick bootstrap
+
+To quickly get a working openQA installation, you can use openQA-bootstrap.
+
+=== Directly on your machine
+
+This should work on openSUSE Leap and openSUSE Tumbleweed and will setup openQA
+on your machine.
+
+[source,sh]
+-------------------------------------------------------------------------------
+zypper in openQA-bootstrap
+/usr/share/openqa/script/openqa-bootstrap
```

Comment:
Could you call this script as post-install action of command specific subpackages?

Reply:
No, because this will just result in an rpm that takes forever to install without giving user feedback as rpm caches the script output until the script returns.
Also this script adds some repos and installs some packages. This wouldn't work due to the already locked rpm database.